src/pudl/transform/ferc1.py

```diff
@@ -2656,30 +2774,6 @@ def apply_xbrl_calculation_fixes(
                             "source_tables": ["plant_in_service_ferc1"],
                         },
                     },
-                    {
-                        "calc_component_to_replace": {},
-                        "calc_component_new": {
-                            "name": "hydraulic_production_plant",
-                            "weight": 1.0,
-                            "source_tables": ["plant_in_service_ferc1"],
-                        },
-                    },
-                    {
-                        "calc_component_to_replace": {},
-                        "calc_component_new": {
-                            "name": "nuclear_production_plant",
-                            "weight": 1.0,
-                            "source_tables": ["plant_in_service_ferc1"],
-                        },
-                    },
-                    {
-                        "calc_component_to_replace": {},
-                        "calc_component_new": {
-                            "name": "steam_production_plant",
-                            "weight": 1.0,
-                            "source_tables": ["plant_in_service_ferc1"],
-                        },
-                    },
```

Comment:
These were part of what seemed to be an incomplete and unnecessary redirect that went around the node gathering all `production_plant` values into a single calculated value.  However it left out some of the elements of that calculated value, and I'm not sure why it seemed to be necessary. It was breaking the tree structure and seemed incorrect so I removed it.